inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -216,8 +216,11 @@ StatusCode CNNNetworkNGraphImpl::addOutput(const std::string& layerName, size_t
                 std::string outputName = layerName;
                 if (layer->outputs().size() != 1) {
                     outputName += "." + std::to_string(outputIndex);
+                } else if (outputIndex >= layer->outputs().size()) {
+                    return DescriptionBuffer(OUT_OF_BOUNDS, resp)
+                    << "port index " << outputIndex << " exceeds layer's outputs which is " << layer->outputs().size();
```

Comment:
"exceeds the number of layer outputs"

Reply:
updated the exception message

Reply:
I didn't mean to remove "which is", sorry for confusion.

Reply:
No, I got it and removed it too..., looks like I've reverted the change at the last minute, will remove it, thanks!

Reply:
It would be OK if it was "the number of layer outputs (1)" - place the number to brackets.